Description: add missing include for kinfo_getproc(3) on kFreeBSD
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: no
Last-Update: 2015-07-13

--- a/src/lib/corelib/tools/processutils.cpp
+++ b/src/lib/corelib/tools/processutils.cpp
@@ -41,6 +41,7 @@
 #   include <unistd.h>
 #   include <cstdio>
 #elif defined(Q_OS_BSD4)
+#   include <libutil.h>
 #   include <sys/user.h>
 #else
 #   error Missing implementation of processNameByPid for this platform.
