Description: disable a test that fails on most architectures
Author: Dmitry Shachnev <mitya57@debian.org>
Bug: https://bugreports.qt.io/browse/QBS-863
Forwarded: no
Last-Update: 2015-08-25

--- a/src/lib/corelib/language/tst_language.cpp
+++ b/src/lib/corelib/language/tst_language.cpp
@@ -492,8 +492,6 @@
                "but this is qbs version " QBS_VERSION ".";
     QTest::newRow("wrongQbsVersionFormat")
             << "The value '.*' of Project.minimumQbsVersion is not a valid version string.";
-    QTest::newRow("properties-item-with-invalid-condition")
-            << "TypeError: Result of expression 'cpp.nonexistingproperty'";
     QTest::newRow("misused-inherited-property") << "Binding to non-item property";
 }
 
