Description: detect endianness of all Debian supported archs
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-yet
Last-Update: 2014-10-31

--- a/src/lib/corelib/tools/architectures.cpp
+++ b/src/lib/corelib/tools/architectures.cpp
@@ -61,6 +61,9 @@
         << QLatin1String("intel64")
         << QLatin1String("mingw64"));
 
+    archMap.insert(QLatin1String("arm64"), QStringList()
+        << QLatin1String("aarch64"));
+
     archMap.insert(QLatin1String("ia64"), QStringList()
         << QLatin1String("ia-64")
         << QLatin1String("itanium"));
@@ -71,6 +74,9 @@
     archMap.insert(QLatin1String("ppc64"), QStringList()
         << QLatin1String("powerpc64"));
 
+    archMap.insert(QLatin1String("ppc64le"), QStringList()
+        << QLatin1String("powerpc64le"));
+
     QMapIterator<QString, QStringList> i(archMap);
     while (i.hasNext()) {
         i.next();
@@ -89,15 +95,27 @@
     QStringList little = QStringList()
             << QLatin1String("x86")
             << QLatin1String("x86_64")
+            << QLatin1String("alpha")
             << QLatin1String("arm")
-            << QLatin1String("arm64");
+            << QLatin1String("arm64")
+            << QLatin1String("mipsel")
+            << QLatin1String("mipsn32el")
+            << QLatin1String("mips64el")
+            << QLatin1String("ppc64le");
 
     if (little.contains(canonicalArch))
         return QLatin1String("little");
 
     QStringList big = QStringList()
+            << QLatin1String("hppa")
+            << QLatin1String("mips")
+            << QLatin1String("mipsn32")
+            << QLatin1String("mips64")
             << QLatin1String("ppc")
-            << QLatin1String("ppc64");
+            << QLatin1String("ppc64")
+            << QLatin1String("s390x")
+            << QLatin1String("sparc")
+            << QLatin1String("sparc64");
 
     if (big.contains(canonicalArch))
         return QLatin1String("big");
