Description: move hostosinfo.h to public API, qtcreator needs it
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-yet
Last-Update: 2014-10-06

--- a/src/lib/corelib/corelib.qbs
+++ b/src/lib/corelib/corelib.qbs
@@ -296,7 +296,6 @@
             "fileinfo.cpp",
             "fileinfo.h",
             "filetime.h",
-            "hostosinfo.h",
             "id.cpp",
             "id.h",
             "installoptions.cpp",
@@ -336,6 +335,7 @@
             "cleanoptions.h",
             "codelocation.h",
             "error.h",
+            "hostosinfo.h",
             "installoptions.h",
             "preferences.h",
             "processresult.h",
