Description: fix the list of installed headers for qmake
Origin: upstream, https://code.qt.io/cgit/qbs/qbs.git/commit/?id=566124f04be2c576
Last-Update: 2018-04-04

--- a/src/lib/corelib/tools/tools.pri
+++ b/src/lib/corelib/tools/tools.pri
@@ -110,21 +110,20 @@ osx {
 !qbs_no_dev_install {
     tools_headers.files = \
         $$PWD/architectures.h \
+        $$PWD/buildoptions.h \
         $$PWD/cleanoptions.h \
         $$PWD/codelocation.h \
         $$PWD/commandechomode.h \
         $$PWD/error.h \
-        $$PWD/settings.h \
-        $$PWD/settingsmodel.h \
+        $$PWD/generateoptions.h \
+        $$PWD/installoptions.h \
         $$PWD/preferences.h \
-        $$PWD/profile.h \
         $$PWD/processresult.h \
+        $$PWD/profile.h \
+        $$PWD/projectgeneratormanager.h \
         $$PWD/qbs_export.h \
-        $$PWD/buildoptions.h \
-        $$PWD/generateoptions.h \
-        $$PWD/generatorpluginmanager.h \
-        $$PWD/installoptions.h \
-        $$PWD/qbspluginmanager.h \
+        $$PWD/settings.h \
+        $$PWD/settingsmodel.h \
         $$PWD/setupprojectparameters.h \
         $$PWD/toolchains.h \
         $$PWD/version.h

