Description: install missing public header
Origin: https://codereview.qt-project.org/199668
Last-Update: 2017-07-10

--- a/src/lib/corelib/tools/tools.pri
+++ b/src/lib/corelib/tools/tools.pri
@@ -125,7 +125,8 @@
         $$PWD/generatorpluginmanager.h \
         $$PWD/installoptions.h \
         $$PWD/setupprojectparameters.h \
-        $$PWD/toolchains.h
+        $$PWD/toolchains.h \
+        $$PWD/version.h
     tools_headers.path = $${QBS_INSTALL_PREFIX}/include/qbs/tools
     INSTALLS += tools_headers
 }
