Description: do not hardcode hash style
Author: Dmitry Shachnev <mitya57@debian.org>
Bug: https://bugreports.qt-project.org/browse/QBS-696
Forwarded: not-needed
Last-Update: 2014-10-08

--- a/share/qbs/modules/cpp/GenericGCC.qbs
+++ b/share/qbs/modules/cpp/GenericGCC.qbs
@@ -121,7 +121,6 @@
             args.push('-shared');
             if (product.moduleProperty("qbs", "targetOS").contains('linux')) {
                 args = args.concat([
-                    '-Wl,--hash-style=gnu',
                     '-Wl,--as-needed',
                     '-Wl,-soname=' + UnixUtils.soname(product, lib.fileName)
                 ]);
