Description: skip homeDirectory test for Debian build
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-needed
Last-Update: 2015-07-18

--- a/src/lib/corelib/language/tst_language.cpp
+++ b/src/lib/corelib/language/tst_language.cpp
@@ -768,6 +768,7 @@
 
 void TestLanguage::homeDirectory()
 {
+    QSKIP("buildds do not have home directory");
     try {
         defaultParameters.setProjectFilePath(testProject("homeDirectory.qbs"));
         ResolvedProjectPtr project = loader->loadProject(defaultParameters);
